.env

```diff
@@ -0,0 +1,7 @@
+COMPOSE_PROJECT_NAME=md
+GIRDER_ADMIN_NAME=admin
+GIRDER_ADMIN_PASS=admin
+GIRDER_USER_NAME=girder
+GIRDER_USER_PASS=girder
+
+MAKE_PARALLELISM=-j4
```

Comment:
I might rename this file to something like `env.example` so that people are less likely to deploy with the default passwords.